packages/cli/src/lib/project/project.spec.ts

```diff
@@ -55,18 +64,42 @@ describe('Project', () => {
     doMockCreateWriteStream();
   });
 
-  beforeEach(() => {
-    doMockInValidProject();
-  });
-
   describe('if the project is invalid', () => {
-    it('should ensure project exists', async () => {
+    it('should ensure resources folder exists', async () => {
+      doMockFileDoesNotExists('resources');
       const project = projectCreator();
       project.compressResources();
       expect(mockedError).toHaveBeenCalledWith(
-        new Error(InvalidProjectError.message)
+        new Error(
+          'dummy/path is not a valid project: Does not contain any resources folder'
+        )
       );
     });
+
+    it('should ensure .coveo hidden folder exists', async () => {
+      doMockFileDoesNotExists('_');
+      const project = projectCreator();
+      doMockFileDoesNotExists('.coveo');
+      project.compressResources();
+      expect(mockedError).toHaveBeenCalledWith(
+        new Error(
+          'dummy/path is not a valid project: Does not contain any .coveo folder'
+        )
+      );
+    });
+  });
+
+  it('should create .coveo project if absent', () => {
+    doMockFileDoesNotExists('.coveo');
+    mockedPathExistsSync.mockReturnValueOnce(false);
+    projectCreator();
+    expect(mockedCreateFileSync).toHaveBeenCalledWith(
+      expect.stringContaining('.coveo')
+    );
+    expect(mockedJSONSync).toHaveBeenCalledWith(
+      expect.stringContaining('.coveo/config.json'),
```

Comment:
```suggestion
      expect.stringContaining(join('.coveo/config.json')),
```

You need this for your test to work on Windows because of the variations in path separator (`/` v.s `\\`)